web/rest/brand/src/Controller/Provider/Administrator/MassUpdateAclsAction.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Controller\Provider\Administrator;
+
+use Ivoz\Provider\Domain\Model\Administrator\AdministratorInterface;
+use Ivoz\Provider\Domain\Model\Administrator\AdministratorRepository;
+use Service\AdministratorRelPublicEntity\SwitchAcl;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class MassUpdateAclsAction
+{
+    public function __construct(
+        private AdministratorRepository $administratorRepository,
+        private SwitchAcl $switchAcl
+    ) {
+    }
+
+    public function __invoke(Request $request): Response
+    {
+        /** @var int[] $publicEntitiesRelUserIds */
+        $publicEntitiesRelUserIds =  $request->toArray();
+        $adminId = (int) $request->attributes->get('id');
+        $routeName = (string) $request->attributes->get('_route');
+
+        /** @var ?AdministratorInterface $administrator */
+        $administrator = $this->administratorRepository->find($adminId);
+
+        if (is_null($administrator)) {
+            return new Response('Administrator not found', 404);
+        }
```

Comment:
Missing:
- Check that administrator is restricted
- You can manage it (Company admin of a company you own)
Forbidden otherwise